server/spec/mutations/stacks/create_spec.rb

```diff
@@ -101,6 +101,22 @@
       expect(outcome.result.stack_revisions.count).to eq(1)
     end
 
+    it 'creates stack labels' do
+      outcome = described_class.new(
+        grid: grid,
+        name: 'stack',
+        stack: 'foo/bar',
+        version: '0.1.0',
+        registry: 'file://',
+        labels: ['fqdn=about.me'],
+        source: '...',
+        variables: {foo: 'bar'},
+        services: [{name: 'redis', image: 'redis:2.8', stateful: true }]
+      ).run
+      expect(outcome).to be_success
+      expect(outcome.result.labels).to eq(['fqdn=about.me'])
```

Comment:
maybe replace these test fqdn:s with something like `example.com` instead of some actual sites.

Reply:
✅ RESOLVED